WooCommerce/Classes/ViewRelated/Orders/Order Creation/CollapsibleProductCard.swift

```diff
@@ -153,13 +153,31 @@ private struct CollapsibleProductRowCard: View {
                         Text(viewModel.productDetailsLabel)
                             .font(.subheadline)
                             .foregroundColor(isCollapsed ? Color(.textSubtle) : Color(.text))
+                        Text(viewModel.subscriptionConditionsDetailsLabel)
+                            .subheadlineStyle()
+                            .renderedIf(viewModel.shouldShowProductSubscriptionsDetails && isCollapsed)
```

Comment:
Maybe we should check if this string is empty?
Like this
```
if !viewModel.subscriptionConditionsDetailsLabel.isEmpty {
 Text(viewModel.subscriptionConditionsDetailsLabel)
  .subheadlineStyle()
  .renderedIf(viewModel.shouldShowProductSubscriptionsDetails && isCollapsed)
}
```
Because it makes an empty visible space if it is an empty string.

Or we can change that `subscriptionConditionsDetailsLabel` is an optional string and in its creation we check if the created string is not empty to return it, otherwise we return nil.
See what best fits you :)

Reply:
Example with and without checking if the string is empty

![IMG_0801](https://github.com/woocommerce/woocommerce-ios/assets/6242034/45546803-5698-4741-9ba0-f1dcd37efc70)

![IMG_0802](https://github.com/woocommerce/woocommerce-ios/assets/6242034/bf0625d5-00eb-43a5-bbe1-3f623cac5c53)

Reply:
Ah good catch! Made the String optional on https://github.com/woocommerce/woocommerce-ios/pull/12454/commits/d4e3a8aa4caac8d3c4a64203ea669e42a3eac4bd so the extra space is removed when nil

Reply:
> Updated on [fdbae26](https://github.com/woocommerce/woocommerce-ios/commit/fdbae26a8b4ebc6df9feade2d0d7eab7dff0f258) to check for empty instead.

Great! :)